securedrop_salt/sd-app.sls

```diff
@@ -14,13 +14,16 @@
 
 include:
   - securedrop_salt.sd-workstation-template
-  - securedrop_salt.sd-upgrade-templates
+  - securedrop_salt.sd-viewer
 
 sd-app:
   qvm.vm:
     - name: sd-app
     - present:
+      # Sets attributes if creating VM for the first time,
+      # otherwise `prefs` must be used.
       - label: yellow
```

Comment:
Should this also be duplicated under `prefs`? Or did you not because we assume it's static (which is reasonable IMO)

Reply:
The colour label you mean? yeah I guess technically someone could change the label colour. (If they do, do we _want_ to forcibly change it back?)

Reply:
Yeah, I was just curious why it was the only key that wasn't duplicated. I think it's a reasonable thing to only set once and let the user override if they want.

But let's explicitly document it, and maybe a test can assert the color is only set in present and not under prefs?

Reply:
Have added explanatory comments (squashed) in https://github.com/freedomofpress/securedrop-workstation/pull/1226/commits/b70de6c7ca9ce1dfeba081f9ddd794bf0819b9c2